devrun.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
I'm unsure about this last script, not the part about merging sbt & webpack (that's perfect) but rather wouldn't it be better to 'teach to fish' ux&d and get them familiar with yarn? we can do a `yarn full-devrun` command to run both using yarn and in turn get them familiar with using the `yarn` prefix